src/sentry/feedback/models.py

```diff
@@ -19,7 +19,11 @@ class Feedback(Model):
     date_added = models.DateTimeField(default=timezone.now)
     organization_id = BoundedBigIntegerField(db_index=True)
     environment = FlexibleForeignKey("sentry.Environment", null=True)
-
+    assignee = FlexibleForeignKey(
+        "sentry.Actor",
+        null=True,
+        on_delete=models.SET_NULL,
+    )
```

Comment:
Elsewhere in the application (notification settings, and external actors) we've been moving away from the actor model and instead using user_id and team_id columns. The actor model adds more complexity to read queries as an additional record needs to be loaded and, then we still need to load the user/team underneath the actor.

Reply:
got it -- I wonder if there's an opportunity to create some kind of `OwnedModel` ABC that encapsulates that along with helper methods for storing retrieving user_id/team_id?

Reply:
> I wonder if there's an opportunity to create some kind of OwnedModel/AssignedModel ABC that encapsulates that along with helper methods for storing and retrieving user_id/team_id?

Yeah, we haven't explored that yet, but it seems like a good solution to wrap up the multiple key columns.